docs/actions/match.md

```diff
@@ -161,6 +161,23 @@ end
 
 Then all your team has to do is `fastlane certificates` and keys, certs and profiles for all targets will be synced.
 
+#### Handle multiple apps per developer/distribution certificate
+
+If you want to use a single developer and/or distribution certificate for multiple apps belonging to the same development team, you may use the same signing identities repository and branch to store the signing identities for your apps:
+
+Matchfile for App #1:
+```no-highlight
+git_url "https://github.com/example/example-repo.git"
+git_branch "master"
```

Comment:
These should use parens, since we recently decided with that style instead:

```
git_url("https://github.com/example/example-repo.git")
git_branch("master")
```